resources/lang/en/events/death/beach.json

```diff
@@ -160,7 +160,7 @@
             "no_body"
         ],
         "weight": 20,
-        "event_text": "m_c found {PRONOUN/m_c/self} staring at an odd shadow in the sea. One distracted paw slipped and {PRONOUN/m_c/subject} fell into the waves, where something large and unseen tangled around {PRONOUN/m_c/object}, dragging {PRONOUN/m_c/object} into the abyss.",
+        "event_text": "m_c fell into the water, something large and unseen dragged {PRONOUN/m_c/object} beneath the waves.",
```

Comment:
```suggestion
        "event_text": "m_c fell into the water, where something large and unseen dragged {PRONOUN/m_c/object} beneath the waves.",
```

Reply:
This is grammatically incorrect without "where". You could put "when" at the front of the sentence and that would also fix it ("when the cat fell in, something dragged it in"), but as it stands it's not valid English I don't think?